### ISSUE TITLE
Add visible indicator that Command+K exists

### ISSUE THREAD

**Issue by metafeather** [aptakube/aptakube#117]
### What problem are you trying to solve? Do you a suggestion on how to solve it?

I was unaware that searching CRD's had been replaced by the functionality in a Command Palette invoked using Command+K.

Whilst this is great functionality it is not discoverable and I suggest a console icon + Command+K text is added next to the Namespaces selector, which can be seen to have a Command+N shortcut.

**Comment by goenning** [comment]
Yeah, it's not great today. But there's also a lot of other shortcuts that are not easily discoverable. I'm considering adding a "Shortcuts / Help" button on the initial screen with a list of all shortcuts, including this one.
